doc/src/tutorials/physics/mechanics/duffing-example.rst

```diff
@@ -0,0 +1,141 @@
+.. _duffing-example:
+
+==================================
+Duffing Oscillator with a Pendulum
+==================================
+
+In this example we demonstrate the use of functionality provided in
+:obj:`sympy.physics.mechanics` for deriving the quations of motion (EOM) for a system
+consisting of a Duffing oscillator with a pendulum.
```

Comment:
Would be nice to refer that this example is inspired by that paper (I think you can use something like `:cite:...`).